src/components/plots/gridPlot.ts

```diff
@@ -96,6 +96,60 @@ export module Plots {
     protected _generateDrawSteps(): Drawers.DrawStep[] {
       return [{attrToProjector: this._generateAttrToProjector(), animator: this._getAnimator("cells")}];
     }
+
+    public x(): Plots.AccessorScaleBinding<any, number>;
+    public x(x: number | _Accessor): Grid;
+    public x(x: any | _Accessor, scale: Scale<any, number>): Grid;
+    public x(x?: number | _Accessor | any, scale?: Scale<any, number>): any {
+      if (x == null) {
+        return super.x();
+      }
+      if (scale == null) {
+        super.x(<number | _Accessor> x);
+      } else {
+        super.x(<any | _Accessor> x, scale);
```

Comment:
You might not need the `if`-`else` any more if the compiler evaluates it correctly.